currency-v1/apiproxy/resources/jsc/ConfigureTargetRequest.js

```diff
@@ -1,3 +1,2 @@
 context.setVariable('target.copy.pathsuffix', false);
-context.setVariable('author', 'nicola');
-
+context.setVariable('author', 'sean');
```

Comment:
looks great - although this seems to be a pointless change.

but sprint ends today so push